momento-sdk/src/main/java/momento/sdk/ISubscribeCallOptions.java

```diff
@@ -0,0 +1,11 @@
+package momento.sdk;
+
+import momento.sdk.responses.topic.TopicMessage;
+
+public interface ISubscribeCallOptions {
```

Comment:
are we able to add the `resumeAtTopicSequenceNumber` to this interface? I am not quite sure how java interfaces work, but we need to add that to the `subscribe` method as well

Reply:
We have it in subscription state. Javascript one does not pass the param in the subscribe method. I was confused on this as well. @cprice404 any suggestions?